ui/core/components/inputs/buffs_debuffs.ts

```diff
@@ -937,6 +947,12 @@ export const RAID_BUFFS_CONFIG = [
 		stats: [Stat.StatArmor],
 		faction: Faction.Alliance,
 	},
+	{
+		config: SanctityAura,
+		picker: IconPicker,
+		stats: [],
```

Comment:
Stat.HolyPower

Reply:
Fixed in https://github.com/wowsims/sod/pull/302/commits/6085426bc9c220f0b23d0f0779006ae7408a9b60